### ISSUE TITLE
Homey is frequently offline

### ISSUE THREAD

**Issue by AJediIAm** [athombv/homey#439]
**What is your Homey version (Settings → System)?**

0.8.29

**What is the problem?**

My Homey goes offline several times per day. I need to pull the power to get it back online. When it is offline, I think it is still performing all it's scheduled tasks, but I cannot reach my Homey via the android app of via my.athom.com. 

When I try to ping the IP adres, i get the response that the host is unreachable. My router does not show the Homey as a connected device. 

**Any other remarks**

I <3 Homey

**Comment by Fieldhome** [comment]
I have the same issue after the 0.8.29 update was installed.

**Comment by thezahir1** [comment]
Same here.... (also after performing a reset to establish a new connection to my home's wifi)

**Comment by djefmo** [comment]
Same here; on 0.8.32

**Comment by WeeJeWel** [comment]
We can reproduce the issue. Logs didn't give us any information, so it's gonna be hard to debug. Mostly because it takes a lot of time.

If anyone can find a way to reproduce it faster, that'd be a great help!

**Comment by Dijker** [comment]
Version 0.8.35
Today after approx 3 days online at 21u12 I was notified:
The monitor Homey (https://56a1e23fbdc38b966486ad16.homey.athom.com/api/manager/webserver/ping) is currently DOWN (HTTP 500 - Internal Server Error).
 {"success":false,"message":"homey_not_connected"}

Homey was spinning, 
At first I got a (Offline) logging on to Homey. 
After talking to Homey ho became online and I tested NFC, tried to install an app from the store got an RED cross: BerliozHomey is unreachable. Try again. 
I tried NFC again and send Logs.
2016-05-25 22u52m07s : B80F90E8AD
After restart it was online again and I could install the App.....
